server/config/express.js

```diff
@@ -43,10 +43,16 @@ exports.configure = function configure(app) {
     resave: Boolean(process.env.SESS_RESAVE),
     saveUninitialized: Boolean(process.env.SESS_UNINITIALIZED),
     unset: process.env.SESS_UNSET,
-    cookie: { secure : true },
+    cookie: { httpOnly : true },
     retries: 50
   }));
 
+  // indicate that we are running behind a trust proxy and should use a secure cookie
```

Comment:
According to the article the sub net or some address should be set even if it is not important so what will be our position we will configure the sub net or not? if yes how? (Notice that we will face many type of network in terms of address configuration)